deploy/bin/update-worker-types.js

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env node
+
+const bluebird = require('bluebird');
```

Comment:
if we do end up using bluebird in the end, I noticed it was not included in package.json.  I would like to first address the need for this library as well as promisfying jsonfile that I mentioned in my other comment.

Reply:
I am using bluebird only because it was already present in `package.json`.

Reply:
hrm, I don't see it here: https://github.com/taskcluster/docker-worker/blob/master/package.json

Reply:
Err, you are right, I must have fooled myself look at one of the `node_modules/*` `package.json`.